### PR TITLE
WIP chore(test): test removal of templates on root level

### PR DESCRIPTION
**Pull request by MaxTru**
__Which issue does this PR address?__

related to camunda/camunda-modeler#2128
requires bpmn-io/diagram-js#540

### CODE REVIEW

**Comment by MaxTru** [comment]
WIP: will work as soon as we have requires bpmn-io/diagram-js#540

File: lib/provider/camunda/element-templates/parts/DescriptionProps.js [hunk 1]
```diff
@@ -90,7 +91,9 @@ module.exports = function(
               eventDefinitionType: eventDefinitionType
             });
 
-            selection.select(newElement);
+            if (!is(newElement, 'bpmn:Process') && !is(newElement, 'bpmn:Collaboration')) {
```
Comment (nikku) [inline lib/provider/camunda/element-templates/parts/DescriptionProps.js:94]: Why is this check necessary? What would we like to achieve? 

General remark: 

* From my point of view, this behavior in general could be part of the [`SelectionBehavior`](https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/selection/SelectionBehavior.js) and not burried here, too.

File: test/spec/provider/camunda/element-templates/parts/DescriptionPropsSpec.js [hunk 2]
```diff
@@ -180,109 +246,153 @@ describe('element-templates/parts - Description Properties', function() {
...
+
+        // then
+        var collaboration = elementRegistry.get('Collaboration_1'),
+            root = canvas.getRootElement(),
+            rootBo = getBusinessObject(root);
+
+        expect(collaboration).not.to.exist;
+
+        expect(root).to.exist;
+        expect(is(root, 'bpmn:Collaboration')).to.be.true;
```
Comment (nikku) [inline test/spec/provider/camunda/element-templates/parts/DescriptionPropsSpec.js:391]: There is a few more things that I'd like to see specified, not here but in bpmn-js most likely:

* Does replace keep the old ID? 
* Does root replace keep existing message and error event definitions?
* Does replace keep existing, unknown extension elements?

File: lib/provider/camunda/element-templates/parts/DescriptionProps.js (hunk 1 above)
Comment (MaxTru) [reply lib/provider/camunda/element-templates/parts/DescriptionProps.js]: good point, see https://github.com/bpmn-io/diagram-js/pull/540/commits/33acbcf9cac50173a4487328580b58489ca2cf80

**Comment by MaxTru** [comment]
See https://github.com/bpmn-io/diagram-js/pull/540#issuecomment-815639997

**Pull request closed without merging**